source/walkthroughs/wt_active_learning.rst

```diff
@@ -0,0 +1,227 @@
+Step-by-Step Guide to Active Learning with Log Data in MindMeld
+===============================================================
+
+Active Learning is an approach to continuously and iteratively pick data that is most informative for the models to train on from a pool of unlabelled data-points. By picking such data-points, active learning provides a higher chance of improving the accuracy of the model with fewer training examples. It also greatly reduces the number of queries to be annotated by humans. MindMeld provides this inbuilt functionality to select these must-have queries from existing data or additional logs and datasets to get the best out of your conversational applications.
+
+The following step-by-step guide will use the :doc:`HR Assistant <../blueprints/hr_assistant>` blueprint for showcasing MindMeld's active learning functionality and the different customizations involved.
+
+Step 1: Setup a MindMeld App with Log Resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step for running active learning is to set up your MindMeld app with log data configured for the app. This log data can either be in the form of labelled text files (similar to the train and test files in the app), or an unlabelled text file containing raw text queries across all domains and intents.
```

Comment:
The American spelling is "labeled" and "unlabeled": "This log data can either be in the form of _labeled_ text files (similar to the train and test files in the app), or an _unlabeled_ text file"